transports/websocket/src/framed.rs

```diff
@@ -489,7 +498,8 @@ impl AsRef<[u8]> for IncomingData {
         match self {
             IncomingData::Binary(d) => d,
             IncomingData::Text(d) => d,
-            IncomingData::Pong(d) => d
+            IncomingData::Pong(d) => d,
+            IncomingData::Closed(_) => unimplemented!(),
```

Comment:
@mxinden this is the exact same situation as above, except it's not even possible to pretend there are any inner bytes to reference.